model/README.md

```diff
@@ -30,6 +33,9 @@ files.
 
 ```bash
 cd model_training
+# export shared modules
+export PYTHONPATH=$PYTHONPATH:../../oasst-shared
```

Comment:
This should not be necessary, normally you simply install the oasst-shared module (see readme there https://github.com/LAION-AI/Open-Assistant/tree/main/oasst-shared ). I will check the dependencies .. if mostly related to data this will be resolved as soon by a oasst-data is module which can be installed via pip ...

Reply:
may we provide this as an option? It seems `ossst-data` is still being updated frequently, so maybe just exporting it instead of installing it should also be ok